### PR TITLE
Add category for Carrefour spiders.

### PR DESCRIPTION
**Pull request by panasonic-mafia**

### CODE REVIEW

File: locations/spiders/carrefour_fr.py [hunk 1]
```diff
@@ -1,6 +1,40 @@
 from locations.categories import Categories, apply_category
+from locations.items import Feature
 from locations.storefinders.woosmap import WoosmapSpider
 
+CARREFOUR_SUPERMARKET = {
+    "brand": "Carrefour",
+    "brand_wikidata": "Q217599",
+    "extras": Categories.SHOP_SUPERMARKET,
```
Comment (davidhicks) [inline locations/spiders/carrefour_fr.py:8]: `.value` is needed as a suffix when using categories in this way.

**Review by davidhicks — CHANGES_REQUESTED** [review]

**Comment by panasonic-mafia** [comment]
Looks like `carrefour_pl` has CDN issues - what can I do about it?

File: locations/spiders/carrefour_fr.py (hunk 1 above)
Comment (panasonic-mafia) [reply locations/spiders/carrefour_fr.py]: I'm using `apply_category()`. The name of the property in brand mappings is misleading, sorry for that, changed it to `category`.

**Comment by davidhicks** [comment]
Thanks for the update, sorry I misread the code and how you were applying categories.

`carrefour_pl` is currently crawling OK for me from Australia. Is your ISP/country being geoblocked?

**Comment by panasonic-mafia** [comment]
Ok, all of those spiders are good now.

**Review by davidhicks — CHANGES_REQUESTED** [review]

**Comment by davidhicks** [comment]
Today I learnt of https://en.wikipedia.org/wiki/Kinmen during the review of `carrefour_tw`. How interesting!

File: locations/spiders/carrefour_pl.py [hunk 2]
```diff
@@ -16,11 +22,11 @@ class CarrefourPLSpider(Spider):
...
-        "Hipermarket": {"brand": "Carrefour", "brand_wikidata": "Q217599"},
-        "Market": {"brand": "Carrefour Market", "brand_wikidata": "Q2689639"},
-        "Express (Zielony)": {"brand": "Carrefour Express", "brand_wikidata": "Q2940190"},
-        "Express (Pomarańczowy)": {"brand": "Carrefour Express", "brand_wikidata": "Q2940190"},
-        "Globi": {"brand": "Globi"},
+        "Express (Zielony)": CARREFOUR_EXPRESS,
+        "Express (Pomarańczowy)": CARREFOUR_EXPRESS,
+        "Hipermarket": CARREFOUR_SUPERMARKET,
+        "Market": CARREFOUR_MARKET,
+        "Globi": {"brand": "Globi", "category": Categories.SHOP_CONVENIENCE.value},
```
[the remark this answers is not in the document]
Comment (panasonic-mafia) [reply locations/spiders/carrefour_pl.py]: Thanks!

**Review by davidhicks — APPROVED** [review]

**Pull request merged**